### ISSUE TITLE
grok api

### ISSUE THREAD

**Issue by kaly9** [a-real-ai/pywinassistant#20]
can you use grok api instead of open ai ?

**Comment by TheJustinCrow** [comment]
Do you mean Groq? Groq is pretty compatible with the openai API, but one major thing is that it doesn't support any vision queries. https://console.groq.com/docs/openai

**Comment by kaly9** [comment]
yes that what i mean, Groq is very fast and free if we found a free vision api or local vision that well be great

**Comment by dot-Justin** [comment]
Potentially Groq running llama3 with Ollama running Llava locally
